server/controllers/getProduct.js

```diff
@@ -0,0 +1,10 @@
+
+const getProduct = (req,res)=>{
+    const {product_id}=req.params;
+    getSingleProduct(product_id)
+    .then(() => res.json({ status: 'scusses' }))
+    .catch((err) => res.json({ status: 'err', error: err }));
```

Comment:
here you get the data from the database then you send a message you don't send the data!